girder/models/setting.py

```diff
@@ -133,7 +139,11 @@ def set(self, key, value):
         else:
             setting['value'] = value
 
-        return self.save(setting)
+        setting = self.save(setting)
+
+        self._get.invalidate(self, key)
```

Comment:
Is there a way to modify an existing cache value, or is `invalidate` the only option?